pkg/features/feature.go

```diff
@@ -156,6 +156,12 @@ var (
 		false,
 		false,
 		true).lockOnInstall()
+	ImperativeApiExtension = newFeature(
+		"imperative-api-extension",
+		"Enable imperative API extension as a k8s aggregation layer as proxy to the kube apiserver",
+		true,
```

Comment:
Default should be `false` for now. To make it true, we need two things:
- An actual store being implemented (From Collie) which isn't merged yet
- RDP so that it works for non-helm installation